permuta/perm.py

```diff
@@ -53,29 +62,31 @@ def __new__(cls, iterable=(), check=False):
         """
         return tuple.__new__(cls, iterable)
 
-    def __init__(self, iterable=(), check=False):
+    def __init__(self, iterable: Iterable = (), check: bool = False) -> None:
```

Comment:
Would it be better to specify `Iterable[int]`, maybe also for `__new__`

Reply:
Agreed